graphics/blender/Portfile

```diff
@@ -5,10 +5,19 @@ PortGroup           cmake 1.1
 
 name                blender
 version             2.83.0
-revision            0
+revision            1
```

Comment:
The `revision` should not be increased due to a license change.

Reply:
I could be wrong, but I don't think changing the license line will force the buildbot builders to kick off another set of builds... but a revbump definitely will.

In addition, this change to the licensing is specifically to try to get the builders to upload built binaries to the public archives. My gut tells me that while I'm trying to get this licensing mess sorted out, that each change should be accompanied by an increase to the revision.

If you still disagree, and you're confident that keeping the revision the same will still result in the buildbot kicking off another set of builds, then I'm perfectly willing to keep the revision number the same. Let me know.

Reply:
Changing the portfile will cause the buildbot to rebuild. The `revision` is only for end users with the port already installed. There is no need for them to re-install it for a license change.

Reply:
Ok, sounds good. I've restored the `revision` number to what it was previously.